docs/data/data-grid/list-view/ListView.js

```diff
@@ -8,8 +8,8 @@ import IconButton from '@mui/material/IconButton';
 import MessageIcon from '@mui/icons-material/Message';
 import ToggleButtonGroup from '@mui/material/ToggleButtonGroup';
 import ToggleButton from '@mui/material/ToggleButton';
-import GridIcon from '@mui/icons-material/GridOn';
-import ListIcon from '@mui/icons-material/TableRowsOutlined';
+import GridViewIcon from '@mui/icons-material/ViewModule';
+import ListViewIcon from '@mui/icons-material/ViewList';
```

Comment:
Drive by icon update, these are more consistent with the other icons we use (filled, not outlined).